mysql-test/suite/sys_vars/t/innodb_buffer_pool_dump_abort_loads.opt

```diff
@@ -0,0 +1,6 @@
+--loose-default-storage-engine=innodb
+--loose-innodb_buffer_pool_load_at_startup=0
+--loose-innodb_buffer_pool_dump_at_shutdown=0
+--loose-innodb-buffer-pool-size=8M
+--loose-innodb-buffer-pool-instances=1
```

Comment:
I think that the default `innodb_buffer_pool_instances=0` will be interpreted as 1 when the buffer pool is small.
I’d prefer to avoid unnecessary references to this parameter, because I am considering to revert back to a single buffer pool in [MDEV-15058](https://jira.mariadb.org/browse/MDEV-15058).